dev/forbidden_from_release_tests/bin/main.dart

```diff
@@ -0,0 +1,268 @@
+import 'dart:io';
```

Comment:
Could you add a test that verifies that this actually warns if a forbidden type is found?

Reply:
Added one to test.dart